hardware_interface/include/hardware_interface/types/hardware_interface_trigger_type.hpp

```diff
@@ -0,0 +1,28 @@
+// Copyright 2017 Open Source Robotics Foundation, Inc.
```

Comment:
```suggestion
// Copyright 2024 ros2_control Development Team
```
just a small adjustment here, you can apply/commit the change directly